pyproject.toml

```
@@ -1,3 +1,45 @@
+[project]
+name = "skore"
+description = "Tooling and assistance for data scientists to \"Own Your Data Science\""
+dynamic = ["version"]
+readme = "README.md"
+maintainers = [
+    {name = "skore developers", email="skore@signal.probabl.ai"},
+]
+dependencies = [
+  "diskcache",
+  "fastapi",
+  "rich",
+  "skops",
+  "uvicorn",
+]
+license = { file = "LICENSE" }
+classifiers=[
+  "Intended Audience :: Science/Research",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python",
+  "Topic :: Software Development",
+  "Topic :: Scientific/Engineering",
+  "Development Status :: 3 - Alpha",
+  "Operating System :: Microsoft :: Windows",
+  "Operating System :: POSIX",
+  "Operating System :: Unix",
+  "Operating System :: MacOS",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12"
+]
+
+[project.urls]
+homepage = "https://probabl.ai"
+source = "https://github.com/probabl-ai/skore"
+download = "https://pypi.org/project/skore/#files"
+tracker = "https://github.com/probabl-ai/skore/issues"
+"release notes" = "https://github.com/probabl-ai/skore/releases"
```

```suggestion
[project.urls]
Homepage = "https://probabl.ai"
Repository = "https://github.com/probabl-ai/skore"
Download = "https://pypi.org/project/skore/#files"
Issues = "https://github.com/probabl-ai/skore/issues"
"Release notes" = "https://github.com/probabl-ai/skore/releases"
```

To look more like https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#urls